modules/oadp-creating-backup-cr.adoc

```diff
@@ -54,9 +54,25 @@ spec:
   - <namespace> <1>
   storageLocation: <velero-sample-1> <2>
   ttl: 720h0m0s
+  labelSelector: <3>
+  - matchLabels:
+      app=<label-1>
```

Comment:
Replaceable values use underscores, these should be `<label_1>`, `<label_2>` and so on